soc/xtensa/intel_adsp/common/ipc.c

```diff
@@ -50,24 +55,31 @@ void z_intel_adsp_ipc_isr(const void *devarg)
 
 		regs->tdr = INTEL_ADSP_IPC_BUSY;
 		if (done && !IS_ENABLED(CONFIG_SOC_INTEL_CAVS_V15)) {
+#ifdef CONFIG_SOC_SERIES_INTEL_ACE1X
+			regs->tda = ACE_IPC_TDA1X_DONE;
+#else
 			regs->tda = INTEL_ADSP_IPC_DONE;
+#endif
```

Comment:
This could potentially be cleaned up further by defining the register and done bit contents in one place and then do all the tda/tdd/ida operations without ifdefs in the code. But there are so few, that I don't think this is needed for this initial PR. The code is understandable in current form and the ifdefs have been in use before this PR (for V1.5).